### ISSUE TITLE
AI feature should be improve asking questions based on that keyword

### ISSUE THREAD

**Issue by shahishree03** [algovengers/MindMate#111]
Limited functionality is there so there should be more engaging with AI.
1 Adding , filter out some keywords based on that asking questions.
2. then generative report based on that

**Comment by shahishree03** [comment]
Assign me this do to add , to improve the AI

**Comment by subharthihazra** [comment]
Can you describe how will you approach it?

**Comment by shahishree03** [comment]
yes , sure , by adding another api for mental health api for symptoms related and on that bases it ask questions to choose from the following will you feeling like that from above any ,and generate report based on that 

for report it should be set limit for prior condition of patient based on previous health condition and symptoms

**Comment by subharthihazra** [comment]
Assigned to you.

**Comment by devanshi00** [comment]
@subharthihazra please assign me this issue. I think it is incomplete I want to work on this.
My technical skills include Node,js Express.Js React.Js Tailwind CSS Javascript Bootstrap Django.Along with this I am a AI ML enthusiast with knowledge of Deep Neural Networks ,Gen -AI ,Tensorflow ,Pytorch.

**Comment by shahishree03** [comment]
Hello @devanshi00   how are you procedding this can you please tell me and please share your discord username
